scene/gui/tab_bar.h

```diff
@@ -107,6 +108,10 @@ class TabBar : public Control {
 	bool scroll_to_selected = true;
 	int tabs_rearrange_group = -1;
 
+	const float DEFAULT_GAMEPAD_EVENT_DELAY_MS = 0.5;
+	const float GAMEPAD_EVENT_REPEAT_RATE_MS = 1.0 / 20;
+	float gamepad_event_delay_ms = DEFAULT_GAMEPAD_EVENT_DELAY_MS;
```

Comment:
This is the third control to implement this behavior and define these values. We should unify it somehow (but this is not necessarily a task for this PR).